recipes/gembs-rs/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set version = "4.0" %}
+
+package:
+  name: gembs-rs
+  version: {{ version }}
+
+source:
+  url: https://github.com/heathsc/gemBS-rs/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: d2e4f844e4a381e5a9087c27ed67751ef2a3a2c2a97b02204ff6a483cad95cc9
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage("gembs-rs", max_pin="x") }}
+
+requirements:
+  build:
+    - make
+    - cmake >3.5
+    - autoconf
+    - automake
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+  host:
+    - pip
+    - python
+    - htslib
+    - openssl
+    - bzip2
+    - zlib
+    - freetype
+    - expat
+    - fontconfig
+    - ncurses
+    - zstd
+    - pkg-config
+    - htslib
+  run:
+    - gem3-mapper
+    - python
+    - matplotlib-base
+    - multiprocess
+    - pigz
+    - openssl
+    - bzip2
+    - zlib
+    - matplotlib-base
+    - multiprocess
+    - samtools
+    - bcftools
+    - bs_call
+    - setuptools
+
+test:
+  commands:
+    - gemBS --version
```

Comment:
We'd need a couple more tests in this case to ensure things are working properly because we have, e.g.,
```sh
# gemBS report 
Error: "Could not find (installation) root directory for gemBS at \"/usr/local/lib/gemBS\".  Use root-dir option or set GEMBS_ROOT environment variable"
```
^-- That one might need a little patch involving https://doc.rust-lang.org/std/env/fn.current_exe.html to make it work with Conda's relocatable packages.
I can put a patch/test together for that.